tests/home_work.spec.js

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+   
+     test('Perform Login', async ({ page }) => {
+      await page.goto('https://www.saucedemo.com/');
+      await page.locator('[data-test="username"]').click();
+      await page.locator('[data-test="username"]').fill('standard_user');
+      await page.locator('[data-test="password"]').click();
+      await page.locator('[data-test="password"]').fill('secret_sauce');
+      await page.locator('[data-test="login-button"]').click();
+      const titleLocator = page.locator('[data-test="title"]');
+      await expect(titleLocator).toContainText('Products');
+      const cartIconLocator = page.locator('[data-test="shopping-cart-link"]');
+      await expect(cartIconLocator).toBeVisible();
+      const productCards = page.locator('[data-test="inventory-item"]');
+      const productCount = await productCards.count();
+      expect(productCount).toBeGreaterThan(1);
+    });
+
+  test ('Test 2 - Add product to the cart', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.locator('[data-test="username"]').fill('standard_user');
+  await page.locator('[data-test="password"]').fill('secret_sauce');
```

Comment:
After you'll make all improvements described above, u can try to store credential in a separate file and then import them into your test and use them wherever you need. 
Hardcoding credentials in the tests is a bad practice (storing them into the file inside the repository is a bad practice too, but for learning purposes its okay)